pkgs/applications/editors/emacs/sources.nix

```diff
@@ -50,12 +50,12 @@ in
 
   emacs29 = import ./generic.nix {
     pname = "emacs";
-    version = "29.1-rc1";
+    version = "29.1";
     variant = "mainline";
     src = fetchFromSavannah {
       repo = "emacs";
-      rev = "29.1-rc1";
-      hash = "sha256-p0lBSKsHrFwYTqO5UVIF/PgiqwdhYQE4oUVcPtd+gsU=";
+      rev = "29.1";
+      hash = "1k6jkyyb9g0dsd4nf93hdf9d6nq7a0sqz26q6ls42c04zi7v9319";
```

Comment:
`nix hash to-sri --type sha256 1k6jkyyb9g0dsd4nf93hdf9d6nq7a0sqz26q6ls42c04zi7v9319`
`sha256-KYy0T/wEMEE0NdiIjzVQB1vTkmtwJGdJ0w28tLyf0sw=`

Reply:
Or:
`nix store prefetch-file "https://git.savannah.gnu.org/cgit/emacs.git/snapshot/emacs-29.1.tar.gz"`
Downloaded 'https://git.savannah.gnu.org/cgit/emacs.git/snapshot/emacs-29.1.tar.gz' to '/nix/store/3j07pifs2kh5yap9vky4vk3z4hfhcxc1-emacs-29.1.tar.gz' (hash '`sha256-KYy0T/wEMEE0NdiIjzVQB1vTkmtwJGdJ0w28tLyf0sw=`').

Reply:
Thank you for the info, definitely felt like I was missing that.

Changed the hash!